client/lib/wporg/index.js

```diff
@@ -37,6 +37,7 @@ function getWporgLocaleCode() {
 
 async function pluginRequest( url, body ) {
 	try {
+		// eslint-disable-next-line no-undef -- Probably: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
```

Comment:
It would be better to add `env: { browser: true }` to `./client/.eslintrc.js`. Does that removes the need to add this exclusion?

Reply:
[`ac46ab7` (#46709)](https://github.com/Automattic/wp-calypso/pull/46709/commits/ac46ab7f2aa93858ab98ee2043081260e4087ce8)